packages/@expo/router-core/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@expo/router-core",
+  "version": "6.0.0-beta.10",
```

Comment:
I don't think we have the tooling to keep versions in sync, so we'd probably just have to start at an arbitrary version. Appending `beta` was more accidentally for `expo-router` too at this stage.

Starting at `0.0.1` or `0.1.0` should be fine